srtcore/queue.cpp

```diff
@@ -501,7 +505,11 @@ void CSndQueue::init(CChannel *c, CTimer *t)
     m_pSndUList->m_pWindowCond = &m_WindowCond;
     m_pSndUList->m_pTimer      = m_pTimer;
 
-    ThreadName tn("SRT:SndQ:worker");
+#if ENABLE_LOGGING
```

Comment:
This is required because in case of group testing you usually have multiple threads for `SRT:SndQ:worker` and you need some method to distinguish them in the logs.